R/testthat.R

```diff
@@ -111,6 +111,10 @@ test_api_basic <- function(board) {
     )
   })
 
+  testthat::test_that("can find board required pkgs", {
+    testthat::expect_snapshot(required_pkgs(board))
```

Comment:
I think a snapshot test is probably the best way to check these new methods across all the boards?